infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -155,6 +156,7 @@ private CorfuRuntime getNewCorfuRuntime() {
         final CorfuRuntime.CorfuRuntimeParameters params =
                 serverContext.getDefaultRuntimeParameters();
         params.setSystemDownHandlerTriggerLimit(SYSTEM_DOWN_HANDLER_TRIGGER_LIMIT);
+        params.setPriorityLevel(PriorityLevel.HIGH);
```

Comment:
This should be moved to the server context, otherwise the orchestrator's client would use the same priority.